### ISSUE TITLE
Tests for twisted.persisted should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7723]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7723|
|Type|enhancement|
|Created|2014-11-27 10:00:05Z|
|Branch|https://github.com/twisted/twisted/tree/persisted-testcase-name-7723|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7723.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d9f/d9fedc14fdda53c271f7b85f7af257e10f7cc1b9/535826f0def9cf637b8e5af7389d7f1b2c721f1b.patch) (514 bytes) - added by eeshangarg on 2014-11-27 10:05:37Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7723 7723
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_persisted_testcase_name_7723 branches/persisted-testcase-name-7723
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1417082405473203 1417082405473203
changetime__1422299835372392 1422299835372392
version__None None
owner__adiroiban adiroiban
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7723.patch​) that changes all of the tests in twisted.persisted to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks & regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

(In [43740]) Branching to persisted-testcase-name-7723.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @adiroiban<br>@adiroiban set status to `new`|
|-|-|

Thanks eeshangarg.

I will work to have this merged.

There was no news file in the patch... but I have created one and sent the branch to buildbot.

If all is ok I think that we can merge it.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43767]) Merge persisted-testcase-name-7723: Rename test cases from twisted.persisted.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7723](https://github.com/twisted/twisted/issues/7723)
